src/index/formatFileStructure/buildGraph/addEdge.ts

```diff
@@ -1,6 +1,7 @@
 import { Graph } from "../shared/Graph";
 
-export function addEdge([start, end]: [string, string], graph: Graph) {
+/** Adds an edge to the graph, adding the start if it doesn't already exist. */
```

Comment:
```suggestion
/** Adds an edge to the graph. */
```

Reply:
I don't think this is even necessary anymore now that the function has been renamed from "addEdge" to "addEdgeToGraph".

I'll remove it completely.